vcsaddons/Polar_1D/Polar_1d.ipynb

```diff
@@ -6,12 +6,9 @@
    "source": [
     "# Polar 1D<a id='top' class=\"tocSkip\"> </a>\n",
     "\n",
+    "This notebook shows the different options available for polar 1-d plots.\n",
```

Comment:
I notice you have '1-d plots' here, but somewhere else you have 'one-d'. I think it should be 1-d everywhere for consistency (or 'one-d', but '1-d' is easier to read I think)